docs/general.md

```diff
@@ -0,0 +1,101 @@
+# `Simulation` base class
+
+We defined a base class named `Simulation`. You can find its Python schema definition in [src/nomad_simulations/general.py](https://github.com/nomad-coe/nomad-schema-plugin-simulation-data/blob/develop/src/nomad_simulations/general.py#L168). In NOMAD, this section will
+appear under the **_data_** section for the _archive_ of each _entry_.
+
+In NOMAD, a set of [base sections](https://nomad-lab.eu/prod/v1/staging/docs/howto/customization/base_sections.html) derived from the [Basic Formal Ontology (BFO)](https://basic-formal-ontology.org/) are defined. We used them to define `Simulation` as an [`Activity`](http://purl.obolibrary.org/obo/BFO_0000015). The UML diagram is:
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="../assets/simulation_base.drawio.png" alt="Simulation base section diagram." width="80%" title="Click to zoom in">
+    </label>
+</div>
+
+The detailed UML diagram of quantities and functions defined for `Simulation` is:
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="../assets/simulation.drawio.png" alt="Simulation quantities and functions UML diagram." width="50%" title="Click to zoom in">
+    </label>
+</div>
+
+We use double inheritance from `EntryData` in order to populate the `data` section in the NOMAD archive. You can read more about the public normalize function function and its purpose in our base classes in the corresponding [documentation page](normalize.md). The private function `set_system_branch_depth()` is related with the `ModelSystem` base class, so we refer to its specific [documentation page](model_system/model_system.md) to know more.
```

Comment:
I would prefer using the name of the page for the link instead of the generic "documentation page":

"We use double inheritance from `EntryData` in order to populate the `data` section in the NOMAD archive. All of the base classes discussed here are subject to the [public normalize function](normalize.md) in NOMAD. The private function `set_system_branch_depth()` is related with the [ModelSystem base class](model_system/model_system.md)."

Check the statement about the normalize function, not sure if this is exactly what you meant, but just an example.